pkg/driverbuilder/docker.go

```diff
@@ -110,6 +108,7 @@ func (bp *DockerBuildProcessor) Start(b *builder.Build) error {
 	if err != nil {
 		return err
 	}
+	cli.NegotiateAPIVersion(context.Background())
```

Comment:
This is great, we can now negotiate Client API version against the server one, so that the client version is greater than server version, client can negotiate a lower operating version to satisfy server needs.